### PR TITLE
a buffered identify UI

### PR DESCRIPTION
**Pull request by maxtaco**
- passes simple tests

### CODE REVIEW

File: go/engine/buffered_identify_ui.go [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+	b.Lock()
+	defer b.Unlock()
+	b.proofChecks = append(b.proofChecks, proofCheck{true, p, l})
+	return b.flush(l.BreaksTracking)
+}
+
+func (b *bufferedIdentifyUI) Confirm(o *keybase1.IdentifyOutcome) (keybase1.ConfirmResult, error) {
+	b.Lock()
+	defer b.Unlock()
+	bt := false
```
Comment (patrickxb) [inline go/engine/buffered_identify_ui.go]: what's the point of `bt`?  it's always false?

File: go/engine/buffered_identify_ui.go [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+}
+
+func (b *bufferedIdentifyUI) Finish() error {
+	b.Lock()
+	defer b.Unlock()
+	if b.suppressed {
+		b.G().Log.Debug("| bufferedIdentifyUI#Finish: suppressed")
+		return nil
+	}
+	b.G().Log.Debug("| bufferedIdentifyUI#Finish: went through to UI")
```
Comment (patrickxb) [inline go/engine/buffered_identify_ui.go]: Call `flush` here?

Comment (maxtaco) [inline go/engine/buffered_identify_ui.go]: it can't hurt, but i don't think it will do anything, since the `flush` in `Confirm` covered this case.

Comment (patrickxb) [inline go/engine/buffered_identify_ui.go]: Ok, no need to add it if Confirm path covers it.

**Comment by patrickxb** [comment]
LGTM

**Pull request merged**